specification/ws/src/main/java/org/kaazing/specification/ws/internal/Functions.java

```diff
@@ -34,6 +34,29 @@
     private static final int MAX_ACCEPTABLE_HEADER_LENGTH = 200;
 
     @Function
+    public static String loginBase64Encoder(String login) {
+        byte[] bytes = login.getBytes();
+        return bytesToString(Base64.encode(bytes));
```

Comment:
You should be able to just do `return new String(Base64.encode(bytes));`, not need for the bytesToString function, which you can remove